src/com/geekhub/hw2/vehicle/Car/Car.java

```diff
@@ -0,0 +1,34 @@
+package com.geekhub.hw2.vehicle.Car;
+
+import com.geekhub.hw2.parts.EnergySource;
+import com.geekhub.hw2.parts.Engine;
+import com.geekhub.hw2.parts.Gear;
+import com.geekhub.hw2.vehicle.Car.carPart.Wheel;
+import com.geekhub.hw2.vehicle.Detail;
+import com.geekhub.hw2.vehicle.Direction;
+import com.geekhub.hw2.vehicle.Vehicle;
+
+/**
+ * Created by COMRADE on 17.10.2015.
+ */
+public class Car extends Vehicle {
+    public Car(EnergySource energySource, Engine engine, Gear gear, Detail wheel) {
+        super(energySource, engine, gear, wheel);
+    }
+
+    @Override
+    public void accelerate() {
+        System.out.println("Car accelerate!");
+        gear.consume(engine.transform(energySource.get(), screw));
```

Comment:
Difficult to read one line statement. Better to separate it on steps. 
Also the final step should be next. Details (some running mechanizm) should consume Force and transform it to some torque.